Formula/passwdqc.rb

```diff
@@ -1,8 +1,14 @@
 class Passwdqc < Formula
   desc "Password/passphrase strength checking and enforcement toolset"
   homepage "https://www.openwall.com/passwdqc/"
-  url "https://www.openwall.com/passwdqc/passwdqc-1.4.0.tar.gz"
-  sha256 "72689c31c34d48349a7c2aab2cf6cf95b8d22818758aba329d5e0ead9f95fc97"
+  url "https://www.openwall.com/passwdqc/passwdqc-2.0.2.tar.gz"
+  sha256 "ff1f505764c020f6a4484b1e0cc4fdbf2e3f71b522926d90b4709104ca0604ab"
+  license "0BSD"
+
+  livecheck do
+    url :homepage
+    regex(/href=.*?passwdqc[._-]v?(\d+(?:\.\d+)+)\.t/i)
+  end
```

Comment:
```suggestion
  livecheck do
    url :homepage
    regex(/href=["']?passwdqc[._-]v?(\d+(?:\.\d+)+)\.t/i)
  end
```
If we use `href=.*?`, versions of `pam_passwdqc` and another data file containing `passwdqc` in the archive name are picked up:
```
➜ brew livecheck -d passwdqc
/usr/local/Homebrew/Library/Homebrew/brew.rb (Formulary::FormulaLoader): loading /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/passwdqc.rb

Formula:          passwdqc
Livecheckable?:   Yes

URL (homepage):   https://www.openwall.com/passwdqc/
Strategy:         PageMatch
Regex:            /href=.*?passwdqc[._-]v?(\d+(?:\.\d+)+)\.t/i

Matched Versions:
2.0.2, 1.0.5, 0.09
passwdqc : 2.0.2 ==> 2.0.2
```